kotlin-analysis-gradle/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/gradle/ExtractGradleDependenciesTest.kt

```diff
@@ -0,0 +1,48 @@
+package org.jetbrains.research.ml.kotlinAnalysis.gradle
+
+import org.jetbrains.research.pluginUtilities.util.Extension
+import org.jetbrains.research.pluginUtilities.util.ParametrizedBaseTest
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import java.io.File
+import java.nio.file.Paths
+
+@RunWith(Parameterized::class)
+open class ExtractGradleDependenciesTest :
+    ParametrizedBaseTest(getResourcesRootPath(::ExtractGradleDependenciesTest)) {
+
+    @JvmField
+    @Parameterized.Parameter(0)
+    var inFile: File? = null
+
+    @JvmField
+    @Parameterized.Parameter(1)
+    var outFile: File? = null
+
+    companion object {
+        @JvmStatic
+        @Parameterized.Parameters(name = "{index}: ({0}, {1})")
+        fun getTestData() = getInAndOutArray(
+            ::ExtractGradleDependenciesTest,
+            inExtension = Extension.EMPTY, outExtension = Extension.TXT
+        )
+    }
+
+    @Test
+    fun testExtractRootGradleFileFromProject() {
```

Comment:
This function is almost the same as the testExtractRootGradleFileFromProject from AndroidProjectTaggingTest. Could you create a class or a common function for them?

Reply:
Have just fix it in my next PR with modules extraction [I'll create it tonight].

Reply:
But it is a bad practice to merge code duplicates into the main branch...